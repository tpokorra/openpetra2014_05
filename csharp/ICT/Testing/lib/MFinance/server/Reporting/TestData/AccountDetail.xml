<?xml version="1.0"?>
<Parameters>
  <Parameter id="xmlfiles" value="eString:Finance\\accountdetail.xml,Finance\\accountdetailcommon.xml,Finance\\finance.xml,common.xml" />
  <Parameter id="currentReport" value="eString:Account Detail" />

  <Parameter id="param_diff_period_i" value="eInteger:0" />
  <Parameter id="param_account_hierarchy_c" value="eString:STANDARD" />
  <Parameter id="param_currency" value="eString:Base" />
  <Parameter id="param_period" value="eString:true" />
  <Parameter id="param_date_checked" value="eBoolean:false" />
  <Parameter id="param_start_period_i" value="eInteger:1" />
  <Parameter id="param_end_period_i" value="eInteger:1" />

  <Parameter id="param_year_i" value="eInteger:0" />
  <Parameter id="param_sortby" value="eString:Account" />
  <Parameter id="param_account_codes" value="eString:*NOTUSED*" />
  <Parameter id="param_account_code_start" value="eString:&quot;0100&quot;" />
  <Parameter id="param_account_code_end" value="eInteger:9800" />
  <Parameter id="param_rgrAccounts" value="eString:AccountRange" />
  <Parameter id="param_cost_centre_codes" value="eString:*NOTUSED*" />
  <Parameter id="param_cost_centre_code_start" value="eString:&quot;0400&quot;" />
  <Parameter id="param_cost_centre_code_end" value="eInteger:9500" />
  <Parameter id="param_rgrCostCentres" value="eString:CostCentreRange" />
  <Parameter id="param_depth" value="eString:standard" />


  <Parameter id="param_quarter" value="eString:false" />
  <Parameter id="param_daterange" value="eString:false" />
  <Parameter id="MaxDisplayColumns" value="eInteger:7" />

  <Parameter id="ColumnPosition" column="0" value="eInteger:0" />
  <Parameter id="ColumnPositionIndented" column="0" value="eInteger:0" />
  <Parameter id="ColumnWidth" column="0" value="eInteger:1" />
  <Parameter id="param_calculation" column="0" value="eString:Transaction Cost Centre" />

  <Parameter id="ColumnPosition" column="1" value="eInteger:1" />
  <Parameter id="ColumnPositionIndented" column="1" value="eInteger:0" />
  <Parameter id="ColumnWidth" column="1" value="eInteger:1" />
  <Parameter id="param_calculation" column="1" value="eString:Transaction Account Code" />

  <Parameter id="ColumnPosition" column="2" value="eInteger:6" />
  <Parameter id="ColumnPositionIndented" column="2" value="eInteger:0" />
  <Parameter id="ColumnWidth" column="2" value="eInteger:2" />
  <Parameter id="param_calculation" column="2" value="eString:Debit" />

  <Parameter id="ColumnPosition" column="3" value="eInteger:8" />
  <Parameter id="ColumnPositionIndented" column="3" value="eInteger:0" />
  <Parameter id="ColumnWidth" column="3" value="eInteger:2" />
  <Parameter id="param_calculation" column="3" value="eString:Credit" />

  <Parameter id="ColumnPosition" column="4" value="eInteger:10" />
  <Parameter id="ColumnPositionIndented" column="4" value="eInteger:0" />
  <Parameter id="ColumnWidth" column="4" value="eInteger:5" />
  <Parameter id="param_calculation" column="4" value="eString:Transaction Narrative" />

  <Parameter id="ColumnPosition" column="5" value="eInteger:15" />
  <Parameter id="ColumnPositionIndented" column="5" value="eInteger:0" />
  <Parameter id="ColumnWidth" column="5" value="eInteger:2" />
  <Parameter id="param_calculation" column="5" value="eString:Start Balance" />

  <Parameter id="ColumnPosition" column="6" value="eInteger:17" />
  <Parameter id="ColumnPositionIndented" column="6" value="eInteger:0" />
  <Parameter id="ColumnWidth" column="6" value="eInteger:2" />
  <Parameter id="param_calculation" column="6" value="eString:End Balance" />

  <Parameter id="param_with_analysis_attributes" value="eString:false" />
</Parameters>