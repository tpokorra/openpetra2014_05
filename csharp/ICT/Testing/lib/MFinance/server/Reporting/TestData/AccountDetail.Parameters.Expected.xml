<?xml version="1.0"?>
<Parameters>
  <Parameter id="xmlfiles" value="eString:Finance\\accountdetail.xml,Finance\\accountdetailcommon.xml,Finance\\finance.xml,common.xml" />
  <Parameter id="currentReport" value="eString:Account Detail" />
  <Parameter id="param_diff_period_i" value="eInteger:0" />
  <Parameter id="param_account_hierarchy_c" value="eString:STANDARD" />
  <Parameter id="param_currency" value="eString:Base" />
  <Parameter id="param_period" value="eString:true" />
  <Parameter id="param_date_checked" value="eBoolean:false" />
  <Parameter id="param_start_period_i" value="eInteger:1" />
  <Parameter id="param_end_period_i" value="eInteger:1" />
  <Parameter id="param_year_i" value="eInteger:0" />
  <Parameter id="param_sortby" value="eString:Account" />
  <Parameter id="param_account_codes" value="eString:*NOTUSED*" />
  <Parameter id="param_account_code_start" value="eString:&quot;0100&quot;" />
  <Parameter id="param_account_code_end" value="eInteger:9800" />
  <Parameter id="param_rgrAccounts" value="eString:AccountRange" />
  <Parameter id="param_cost_centre_codes" value="eString:*NOTUSED*" />
  <Parameter id="param_cost_centre_code_start" value="eString:&quot;0400&quot;" />
  <Parameter id="param_cost_centre_code_end" value="eInteger:9500" />
  <Parameter id="param_rgrCostCentres" value="eString:CostCentreRange" />
  <Parameter id="param_depth" value="eString:standard" />
  <Parameter id="param_quarter" value="eString:false" />
  <Parameter id="param_daterange" value="eString:false" />
  <Parameter id="MaxDisplayColumns" value="eInteger:7" />
  <Parameter id="ColumnPosition" column="0" value="eDecimal:0" />
  <Parameter id="ColumnPositionIndented" column="0" value="eInteger:0" />
  <Parameter id="ColumnWidth" column="0" value="eInteger:1" />
  <Parameter id="param_calculation" column="0" value="eString:Transaction Cost Centre" />
  <Parameter id="ColumnPosition" column="1" value="eDecimal:4607182418800017408" />
  <Parameter id="ColumnPositionIndented" column="1" value="eInteger:0" />
  <Parameter id="ColumnWidth" column="1" value="eInteger:1" />
  <Parameter id="param_calculation" column="1" value="eString:Transaction Account Code" />
  <Parameter id="ColumnPosition" column="2" value="eDecimal:4618441417868443648" />
  <Parameter id="ColumnPositionIndented" column="2" value="eInteger:0" />
  <Parameter id="ColumnWidth" column="2" value="eInteger:2" />
  <Parameter id="param_calculation" column="2" value="eString:Debit" />
  <Parameter id="ColumnPosition" column="3" value="eDecimal:4620693217682128896" />
  <Parameter id="ColumnPositionIndented" column="3" value="eInteger:0" />
  <Parameter id="ColumnWidth" column="3" value="eInteger:2" />
  <Parameter id="param_calculation" column="3" value="eString:Credit" />
  <Parameter id="ColumnPosition" column="4" value="eDecimal:4621819117588971520" />
  <Parameter id="ColumnPositionIndented" column="4" value="eInteger:0" />
  <Parameter id="ColumnWidth" column="4" value="eInteger:5" />
  <Parameter id="param_calculation" column="4" value="eString:Transaction Narrative" />
  <Parameter id="ColumnPosition" column="5" value="eDecimal:4624633867356078080" />
  <Parameter id="ColumnPositionIndented" column="5" value="eInteger:0" />
  <Parameter id="ColumnWidth" column="5" value="eInteger:2" />
  <Parameter id="param_calculation" column="5" value="eString:Start Balance" />
  <Parameter id="ColumnPosition" column="6" value="eDecimal:4625478292286210048" />
  <Parameter id="ColumnPositionIndented" column="6" value="eInteger:0" />
  <Parameter id="ColumnWidth" column="6" value="eInteger:2" />
  <Parameter id="param_calculation" column="6" value="eString:End Balance" />
  <Parameter id="param_with_analysis_attributes" value="eString:false" />
  <Parameter id="param_ledger_number_i" value="eInteger:{ledgernumber}" />
  <Parameter id="param_start_date" value="eDateTime:&quot;2013-01-01T00:00:00&quot;" />
  <Parameter id="param_end_date" value="eDateTime:&quot;2013-01-31T00:00:00&quot;" />
  <Parameter id="ColumnFormat" column="0" value="eString:text" />
  <Parameter id="ColumnFormat" column="1" value="eString:text" />
  <Parameter id="ColumnFormat" column="2" value="eString:#,##0.00; ; ; ;" />
  <Parameter id="ColumnFormat" column="3" value="eString: ; #,##0.00; ; ;" />
  <Parameter id="ColumnFormat" column="4" value="eString:text" />
  <Parameter id="ColumnFormat" column="5" value="eString:nothing" />
  <Parameter id="ColumnFormat" column="6" value="eString:nothing" />
  <Parameter id="ColumnPosition" column="112" value="eDecimal:4613937818241073152" />
  <Parameter id="ColumnWidth" column="112" value="eDecimal:4619567317775286272" />
  <Parameter id="ControlSource" column="112" value="eString:Details" />
  <Parameter id="ControlSource" column="100" value="eString:Account Detail" />
  <Parameter id="ControlSource" column="101" value="eString:NUnit Test Ledger {ledgernumber}" />
  <Parameter id="ControlSource" column="106" value="eComposite::&quot;eString:Ledger |eInteger:{ledgernumber}|eString:   NUnit Test Ledger {ledgernumber}&quot;" />
  <Parameter id="ColumnAlign" column="104" value="eString:right" />
  <Parameter id="ControlSource" column="104" value="eComposite::&quot;&quot;&quot;eString:&quot;&quot;&quot;&quot;    Cost Centre: 0400 To &quot;&quot;&quot;&quot;&quot;&quot;|eInteger:9500&quot;" />
  <Parameter id="ColumnAlign" column="105" value="eString:right" />
  <Parameter id="ControlSource" column="105" value="eComposite::&quot;&quot;&quot;eString:&quot;&quot;&quot;&quot;Account: 0100 To &quot;&quot;&quot;&quot;&quot;&quot;|eInteger:9800&quot;" />
  <Parameter id="ColumnWidth" column="107" value="eDecimal:4619567317775286272" />
  <Parameter id="ControlSource" column="107" value="eString:&quot;Currency: EUR Ordered by: Account&quot;" />
  <Parameter id="ControlSource" column="108" value="eComposite::&quot;eString:Current Period is |eInteger:1&quot;" />
  <Parameter id="ControlSource" column="103" value="eComposite::&quot;&quot;&quot;eString:&quot;&quot;&quot;&quot;Period:  &quot;&quot;&quot;&quot;&quot;&quot;|&quot;&quot;eDateTime:&quot;&quot;&quot;&quot;2013-01-01T00:00:00&quot;&quot;&quot;&quot;&quot;&quot;|eString: To |&quot;&quot;eDateTime:&quot;&quot;&quot;&quot;2013-01-31T00:00:00&quot;&quot;&quot;&quot;&quot;&quot;|eString:   (|eInteger:1|eString: To |eInteger:1|eString:)    (CURRENT)&quot;" />
  <Parameter id="ColumnAlign" column="0" value="eString:left" />
  <Parameter id="ColumnCaption" column="0" value="eString:text:CC" />
  <Parameter id="ColumnAlign" column="1" value="eString:left" />
  <Parameter id="ColumnCaption" column="1" value="eString:text:Acc" />
  <Parameter id="ColumnAlign" column="2" value="eString:right" />
  <Parameter id="ColumnCaption" column="2" value="eString:text:Debits" />
  <Parameter id="ColumnAlign" column="3" value="eString:right" />
  <Parameter id="ColumnCaption" column="3" value="eString:text:Credits" />
  <Parameter id="ColumnAlign" column="4" value="eString:left" />
  <Parameter id="ColumnCaption" column="4" value="eString:text:Narrative" />
  <Parameter id="ColumnAlign" column="5" value="eString:left" />
  <Parameter id="ColumnCaption" column="5" value="eString:text:Start Balance" />
  <Parameter id="ColumnAlign" column="6" value="eString:left" />
  <Parameter id="ColumnCaption" column="6" value="eString:text:End Balance" />
  <Parameter id="ColumnAlign" column="-10" level="7" subreport="0" value="eString:right" />
  <Parameter id="ColumnPosition" column="-10" level="7" subreport="0" value="eDecimal:4612361558371493478" />
  <Parameter id="ColumnWidth" column="-10" level="7" subreport="0" value="eDecimal:4616189618054758400" />
  <Parameter id="ColumnAlign" column="-14" level="6" subreport="0" value="eString:left" />
  <Parameter id="ColumnPosition" column="-14" level="6" subreport="0" value="eDecimal:4613937818241073152" />
  <Parameter id="ColumnWidth" column="-14" level="6" subreport="0" value="eDecimal:4620693217682128896" />
  <Parameter id="ColumnFormat" column="2" level="6" subreport="0" value="eString:#,##0.00;#,##0.00;0.00;0.00" />
  <Parameter id="ColumnFormat" column="3" level="6" subreport="0" value="eString:#,##0.00;#,##0.00;0.00;0.00" />
  <Parameter id="ColumnFormat" column="5" level="6" subreport="0" value="eString:#,##0.00 CR;#,##0.00 DR;0.00; " />
  <Parameter id="ColumnFormat" column="6" level="6" subreport="0" value="eString:#,##0.00 CR;#,##0.00 DR;0.00; " />
  <Parameter id="ColumnAlign" column="-10" level="6" subreport="0" value="eString:right" />
  <Parameter id="ColumnPosition" column="-10" level="6" subreport="0" value="eDecimal:4612361558371493478" />
  <Parameter id="ColumnWidth" column="-10" level="6" subreport="0" value="eDecimal:4616189618054758400" />
  <Parameter id="headerVISIBLE" level="6" subreport="0" value="eBoolean:true" />
  <Parameter id="SpaceLineBelow" level="5" subreport="0" value="eBoolean:true" />
  <Parameter id="ColumnAlign" column="-10" level="5" subreport="0" value="eString:right" />
  <Parameter id="ColumnPosition" column="-10" level="5" subreport="0" value="eDecimal:4612361558371493478" />
  <Parameter id="ColumnWidth" column="-10" level="5" subreport="0" value="eDecimal:4616189618054758400" />
  <Parameter id="ColumnFormat" column="2" level="5" subreport="0" value="eString:#,##0.00; ; ; " />
  <Parameter id="ColumnFormat" column="3" level="5" subreport="0" value="eString: ;#,##0.00;0.00; " />
  <Parameter id="ColumnFormat" column="5" level="4" subreport="0" value="eString:#,##0.00 CR;#,##0.00 DR;0.00; " />
  <Parameter id="ColumnFormat" column="6" level="4" subreport="0" value="eString:#,##0.00 CR;#,##0.00 DR;0.00; " />
  <Parameter id="ColumnAlign" column="-10" level="4" subreport="0" value="eString:right" />
  <Parameter id="ColumnPosition" column="-10" level="4" subreport="0" value="eDecimal:4612361558371493478" />
  <Parameter id="ColumnWidth" column="-10" level="4" subreport="0" value="eDecimal:4616189618054758400" />
  <Parameter id="ColumnFormat" column="2" level="4" subreport="0" value="eString:#,##0.00;#,##0.00;0.00;0.00" />
  <Parameter id="ColumnFormat" column="3" level="4" subreport="0" value="eString:#,##0.00;#,##0.00;0.00;0.00" />
  <Parameter id="FullLineBelow" level="3" subreport="0" value="eBoolean:true" />
  <Parameter id="ColumnAlign" column="-10" level="3" subreport="0" value="eString:right" />
  <Parameter id="ColumnPosition" column="-10" level="3" subreport="0" value="eDecimal:4612361558371493478" />
  <Parameter id="ColumnWidth" column="-10" level="3" subreport="0" value="eDecimal:4616189618054758400" />
  <Parameter id="ColumnFormat" column="2" level="3" subreport="0" value="eString:#,##0.00; ; ; " />
  <Parameter id="ColumnFormat" column="3" level="3" subreport="0" value="eString: ;#,##0.00;0.00; " />
  <Parameter id="FullLineAbove" level="2" subreport="0" value="eBoolean:true" />
  <Parameter id="ColumnFormat" column="5" level="2" subreport="0" value="eString:#,##0.00 CR;#,##0.00 DR;0.00; " />
  <Parameter id="ColumnFormat" column="6" level="2" subreport="0" value="eString:#,##0.00 CR;#,##0.00 DR;0.00; " />
  <Parameter id="ColumnAlign" column="-10" level="2" subreport="0" value="eString:left" />
  <Parameter id="ColumnPosition" column="-10" level="2" subreport="0" value="eDecimal:0" />
  <Parameter id="ColumnWidth" column="-10" level="2" subreport="0" value="eDecimal:4611686018427387904" />
  <Parameter id="ColumnFormat" column="2" level="2" subreport="0" value="eString:#,##0.00;#,##0.00;0.00;0.00" />
  <Parameter id="ColumnFormat" column="3" level="2" subreport="0" value="eString:#,##0.00;#,##0.00;0.00;0.00" />
  <Parameter id="ColumnAlign" column="-10" level="1" subreport="0" value="eString:left" />
  <Parameter id="ColumnPosition" column="-10" level="1" subreport="0" value="eDecimal:0" />
  <Parameter id="ColumnWidth" column="-10" level="1" subreport="0" value="eDecimal:4611686018427387904" />
  <Parameter id="ColumnFormat" column="2" level="1" subreport="0" value="eString:#,##0.00; ; ; " />
  <Parameter id="indented" column="2" level="1" subreport="0" value="eBoolean:true" />
  <Parameter id="ColumnFormat" column="3" level="1" subreport="0" value="eString: ;#,##0.00;0.00; " />
  <Parameter id="lowestLevel" level="-2" value="eInteger:7" />
  <Parameter id="ReportWidth" value="eDecimal:4626041242239631360" />
  <Parameter id="ControlSource" column="113" value="eString:Left2" />
</Parameters>